json/aerospike/7.1.0.json

```
@@ -0,0 +1,3557 @@
+                  "maximum": 100,
+                  "description": "",
+                  "dynamic": true,
+                  "enterpriseOnly": true
+                },
+                "mounts": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "",
+                  "dynamic": false,
+                  "enterpriseOnly": true,
+                  "default": []
+                },
+                "mounts-budget": {
+                  "type": "integer",
+                  "default": 0,
+                  "minimum": 1073741824,
+                  "maximum": 18446744073709551615,
+                  "description": "",
+                  "dynamic": true,
+                  "enterpriseOnly": true
+                }
+              }
+            }, {
+              "type": "object",
+              "additionalProperties": false,
+              "enterpriseOnly": true,
+              "required": ["type", "mounts", "mounts-budget"],
+              "properties": {
+                "type": {
+                  "type": "string",
+                  "description": "",
+                  "dynamic": false,
+                  "enterpriseOnly": true,
+                  "default": "flash",
+                  "enum": ["flash"]
+                },
+                "evict-mounts-pct": {
+                  "type": "integer",
+                  "default": 0,
+                  "minimum": 0,
+                  "maximum": 100,
+                  "description": "",
+                  "dynamic": true,
+                  "enterpriseOnly": true
+                },
+                "mounts": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "",
+                  "dynamic": false,
+                  "enterpriseOnly": true,
+                  "default": []
+                },
+                "mounts-budget": {
+                  "type": "integer",
+                  "default": 0,
+                  "minimum": 4294967296,
+                  "maximum": 18446744073709551615,
+                  "description": "",
+                  "dynamic": true,
+                  "enterpriseOnly": true
+                }
+              }
+            }]
+          },
+          "sets": {
+            "type": "array",
+            "enterpriseOnly": false,
+            "items": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "default": " ",
+                  "description": "",
+                  "dynamic": false,
+                  "enterpriseOnly": false
+                },
+                "default-read-touch-ttl-pct": {
+                  "type": "integer",
+                  "default": 0,
+                  "minimum": 0,
```

```suggestion
                  "minimum": -1,
```
The minimum value at the set level is -1. See cfg.c line 3824 https://github.com/citrusleaf/aerospike-server/compare/7.0.0.7...7.1.0.0-rc2#diff-0b637b2046b4f0541a331f7703c763756b27b54f4cff3c8a53f77ecdda57c124R3823

---

The schemas represent a valid aerospike.conf file. Internally, the server stores `-1` to indicate that `default-read-touch-ttl-pct` was not set by the user. `-1` is not a valid value in the aerospike.conf file OR an info call

---

nvm, you are right